learning/tour-of-beam/terraform/cloud_functions/output.tf

```diff
@@ -0,0 +1,20 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+output "cloud-function-trigger-url" {
+  value = google_cloudfunctions_function.cloud_function.*.https_trigger_url
```

Comment:
A description might be valuable in documenting what this output is.  https://developer.hashicorp.com/terraform/language/values/outputs#description-output-value-documentation

Reply:
This file has been removed due, output data is not of importance
But, just for the record, all the resources have descriptions now